app/Http/Controllers/Timelines/TimelineEraController.php

```diff
@@ -219,4 +219,15 @@ protected function datagridAjax($rows)
 
         return response()->json($data);
     }
+
+    /**
+     */
+    public function positionList(Timeline $timeline, TimelineEra $era)
+    {
+        $this->authorize('view', $timeline);
+
+        return response()->json([
```

Comment:
No check that the timeline_era belongs to the timeline, so I can go and load eras from other campaigns just by changing the ID 🙈